modules/oxide/src/less/theme/content/resize-handles/resize-handle.less

```diff
@@ -52,6 +52,11 @@
     outline: 1px dashed black;
     position: absolute;
     z-index: 10000;
+
+    &.mce-resizetable-columns th,
```

Comment:
This was added, as having the internal borders on the ghost image is misleading when using the `resizetable` setting, as only the last column will change size. We probably could update the ghost, but we'd need snooker to take over the resizing entirely before we could do that. Additionally, that'd add extra computation as calculating all the sizes for each mouse movement wouldn't be very cheap.

Reply:
@lnewson The only thing I would add is a comment in the less file what it's used for. Something like 
> Hide table borders when using `table_column_resizing: 'resizetable'`.

Reply:
Good idea, I'll do that now :+1: